crates/router_env/src/env.rs

```diff
@@ -1,21 +1,36 @@
-//!
-//! Current environment related stuff.
-//!
+//! Information about the current environment.
 
 use std::path::PathBuf;
 
 use serde::{Deserialize, Serialize};
 use strum::{Display, EnumString};
 
-/// Parent dir where Cargo.toml is stored
-pub const CARGO_MANIFEST_DIR: &str = "CARGO_MANIFEST_DIR";
-/// Env variable that sets Development/Production env
-pub const RUN_ENV: &str = "RUN_ENV";
+/// Environment variables accessed by the application. This module aims to be the source of truth
+/// containing all environment variable that the application accesses.
+pub mod vars {
+    /// Parent directory where `Cargo.toml` is stored.
+    pub const CARGO_MANIFEST_DIR: &str = "CARGO_MANIFEST_DIR";
 
-///
-/// Current environment.
-///
+    /// Environment variable that sets Development/Sandbox/Production environment.
+    pub const RUN_ENV: &str = "RUN_ENV";
+
+    /// Directory of config TOML files. Default is `config`.
+    pub const CONFIG_DIR: &str = "CONFIG_DIR";
 
+    /// Environment variable that sets the console logging level.
+    pub const RUST_CONSOLE_LOG: &str = "RUST_CONSOLE_LOG";
+
+    /// Environment variable that sets the file logging level.
+    pub const RUST_FILE_LOG: &str = "RUST_FILE_LOG";
+
+    /// Target to which the OpenTelemetry exporter is going to send signals.
+    pub const OTEL_EXPORTER_OTLP_ENDPOINT: &str = opentelemetry_otlp::OTEL_EXPORTER_OTLP_ENDPOINT;
+
+    /// Max waiting time for the OpenTelemetry backend to process each signal batch.
+    pub const OTEL_EXPORTER_OTLP_TIMEOUT: &str = opentelemetry_otlp::OTEL_EXPORTER_OTLP_TIMEOUT;
```

Comment:
For now, I've kept them as environment variables independent of our application configuration. I can update the code to pick these up from application configuration instead (then the `ROUTER__LOG__*` environment variables would have to be set instead), if required.

Reply:
Done, moved to application configuration.